### DISCUSSION TITLE
The trace_id can be found in the logs, but it cannot be queried in Skywalking Trace.

### DISCUSSION THREAD

**Discussion by linshenfanying** [apache/skywalking#12273] [General]
The Skywalking version is 8.9.0, and the Java Agent version is 8.12. It is observed that Trace IDs are generated in the logs, but the corresponding Trace IDs cannot be found in Skywalking, and neither are they available in Elasticsearch.

**Comment by wu-sheng** [comment]
Are you printing Trace IDs or reporting logs to OAP? Those are different features.

**Reply by linshenfanying** [reply]
I am printing Trace IDs and using them to query related traces. I do not have the functionality to report logs.

**Reply by wu-sheng** [reply]
> the corresponding Trace IDs cannot be found in Skywalking, and neither are they available in Elasticsearch.

Then what do you mean by this? **Trace IDs are generated in the logs,** what expected is done.

**Reply by linshenfanying** [reply]
I want to query related trace information on the trace page using the trace ID.

**Reply by wu-sheng** [reply]
That has nothing about the logs. Traces are there if your traces get collected correctly.

**Reply by linshenfanying** [reply]
What has been confirmed is that the same call occurs at different times, and sometimes the trace ID can be used to find the corresponding trace, while other times it cannot.

**Reply by wu-sheng** [reply]
Then it means some plugins are not working as expected, so, some tracing context leaked, you would not see those traces.

You need to address which plugin and why by yourself.

**Reply by linshenfanying** [reply]
what kind of plugins would have such an effect?

**Reply by wu-sheng** [reply]
All tracing plugins could. This is incompatible scenario, which SkyWalking don't support.

**Reply by linshenfanying** [reply]
Thanks。
